internal/service/alertconfiguration/model_test.go

```diff
@@ -248,6 +248,40 @@ func TestAlertConfigurationSDKToTFModel(t *testing.T) {
 				Enabled:               types.BoolValue(true),
 			},
 		},
+		{
+			name: "Complete SKD response with SeverityOverride",
```

Comment:
SDK? there are other places in this file where I spotted this, could you please fix it?
```suggestion
			name: "Complete SDK response with SeverityOverride",
```